modules/docs/arrow-docs/docs/docs/datatypes/option/README.md

```diff
@@ -121,6 +121,13 @@ Arrow also adds syntax to all datatypes so you can easily lift them into the con
 none<String>()
 ```
 
+```kotlin:ank
```

Comment:
Ank is our snippet compiler. It outputs the `toString()` of the value returned by the snippet. You can either remove `:ank` and hardcode the results as they are now, or split it into 2 snippets and let our Ank processor compile them ;)

Reply:
Done :+1: